### ISSUE TITLE
Built-in Currencies

### ISSUE THREAD

**Issue by awhitford** [noojee/money.dart#8]
Rather than [documentation that tells me how to register common currencies](https://pub.dev/packages/money2#common-currencies), why not have these "built-in"?  I know the documentation says:
```dart
/// Money2 does not create a default set of currencies instead you need to explicitly
/// create each currency.
```
but why?  Why not support the currencies mentioned _out of the box_?  We could always register additional currencies if we need.

Alternatively, if you don't want to pre-register, why not simply move the currency list that is documented into code, and the library user can have the convenient option to call [Currencies.registerList](https://pub.dev/documentation/money2/latest/money2/Currencies/registerList.html) against the collection?

**Comment by awhitford** [comment]
BTW, there is a [minor copy/paste bug](https://github.com/noojee/money.dart#common-currencies) for **NZD**.

**Comment by bsutton** [comment]
Thanks for the feedback.

I've just pushed 1.4.0-beta-1 which now has a CommonCurrencies class.

I would be great if you could take a look at the beta and provide any feedback before I do an actual release.

**Comment by bsutton** [comment]
I've just pushed beta2 as I forgot to export the CommonCurrencies class :<

**Comment by bsutton** [comment]
Resolved in 1.4

**Comment by awhitford** [comment]
I appreciate the update on this.
